flycheck.el

```diff
@@ -7512,6 +7512,20 @@ See URL `https://github.com/eslint/eslint'."
                                    (flycheck-error-message err))))
                           (flycheck-sanitize-errors errors))
                   errors)
+  :enabled (lambda (checker)
+             (let* ((command (concat (flycheck-find-checker-executable checker)
+                                     (if flycheck-eslintrc
+                                         (concat " --config " flycheck-eslintrc)
+                                       "")
+                                     " --print-config "
+                                     (shell-quote-argument
+                                      (flycheck-compute-working-directory
+                                       checker))
+                                     "."))
```

Comment:
Why this argument?  Don't we pass the working directory in the previous argument?